python/cuml/tests/test_pickle.py

```diff
@@ -229,13 +229,20 @@ def create_mod():
         if "LogisticRegression" in keys and nrows == 500000:
             nrows, ncols, n_info = (nrows // 20, ncols // 20, n_info // 20)
 
-        X_train, y_train, X_test = make_dataset(datatype, nrows, ncols, n_info)
         if "MBSGD" in keys:
             model = regression_models[keys](
                 fit_intercept=fit_intercept, batch_size=nrows / 100
             )
         else:
             model = regression_models[keys](fit_intercept=fit_intercept)
+        if is_classifier(model):
+            X_train, y_train, X_test = make_classification_dataset(
+                datatype, nrows, ncols, n_info, 2
+            )
```

Comment:
Same as `test_input_estimators`, before we were testing all estimators with regression data, which now errors nicely (matching sklearn behavior) for classifiers. Tests updated to test classifiers with classification data.